packages/console/src/components/Dropdown/DropdownItem.module.scss

```diff
@@ -1,12 +1,14 @@
 @use '@/scss/underscore' as _;
 
 .item {
-  padding: _.unit(2) _.unit(4);
+  padding: 6px _.unit(2);
```

Comment:
nit: mix absolute number and unit looks odd

Reply:
It does. If this is by design, then maybe we can consider using `align-items: center` to center align the inner input

Reply:
It is by design. And it already has `align-items: center;`, I don't think is good to achieve the `6px` padding by other ways, they may have side effects.

Reply:
Discussed and we have aligned the usage of `_.unit`. A `.5` step would be fine